schema/defs-linux.json

```diff
@@ -203,7 +204,8 @@
                     "$ref": "#/definitions/Minor"
                 },
                 "access": {
-                    "type": "string"
+                    "type": "string",
+                    "pattern": "^[rwm]{1,3}$"
```

Comment:
> … but it seems json schema does not support ?!.

JSON Schema [supports the ECMA 262 regular expression dialect][1], which [includes `(?!…)`][2], and your proposed regexp looks good to me.  Maybe just a limitation of gojsonschema?  It looks like it chokes on that pattern with:

    invalid character '1' in string escape code

probably because it's [using Go's stock regexp package][3], which [does not support `(?!…)`][4].  Some related discussion in xeipuuv/gojsonschema#93.

So I'm fine with your current, loose regexp.  If/when gojsonschema gains support for `(?!…)`, we can move to your more-specific regexp.

[1]: https://tools.ietf.org/html/draft-wright-json-schema-validation-00#section-5.8
[2]: http://www.ecma-international.org/ecma-262/5.1/#sec-15.10.1
[3]: https://github.com/xeipuuv/gojsonschema/blob/6b67b3fab74d992bd07f72550006ab2c6907c416/validation.go#L660
[4]: https://github.com/google/re2/wiki/Syntax